packages/components/cascader/src/index.vue

```diff
@@ -726,9 +726,15 @@ export default defineComponent({
 
     watch(presentText, syncPresentTextValue, { immediate: true })
 
+    const getInitialHeight = () => {
+      if (props.size === 'large') return 38
+      if (props.size === 'small') return 22
+      return 30
```

Comment:
I think we should use css variable to replace hardcode here
#9982 #10308

Reply:
His original logic also has inputInner.offsetHeight
 `inputInitialHeight = inputInner.offsetHeight || initialHeight`
cssVar isn't it an another form hardcode
Just personal feeling, there is more burden to find dom to inject style, I don't think it seems to have any advantages over the current method